code/datums/traits/negative.dm

```diff
@@ -0,0 +1,180 @@
+		if("r_arm")
+			prosthetic = new/obj/item/bodypart/r_arm/robot/surplus(trait_holder)
+		if("l_leg")
+			prosthetic = new/obj/item/bodypart/l_leg/robot/surplus(trait_holder)
+		if("r_leg")
+			prosthetic = new/obj/item/bodypart/r_leg/robot/surplus(trait_holder)
+	prosthetic.replace_limb(H)
+	qdel(old_part)
+
+
+
+/datum/trait/insanity
+	name = "Reality Dissociation Syndrome"
+	desc = "You suffer from a severe disorder that causes very vivid hallucinations. Mindbreaker toxin can suppress its effects, and you are immune to mindbreaker's hallucinogenic properties."
+	value = -2
+	//no mob trait because it's handled uniquely
+	gain_text = "<span class='userdanger'>...</span>"
+	lose_text = "<span class='notice'>You feel in tune with the world again.</span>"
+	medical_record_text = "Patient suffers from acute Reality Dissociation Syndrome and experiences vivid hallucinations."
+
+/datum/trait/insanity/add()
+	addtimer(CALLBACK(src, .proc/disclaimer), 50)
+
+/datum/trait/insanity/process()
+	if(trait_holder.reagents.has_reagent("mindbreaker"))
+		trait_holder.hallucination = 0
+		return
+	if(prob(1)) //we'll all be mad soon enough
+		madness()
+
+/datum/trait/insanity/proc/madness(mad_fools)
+	set waitfor = FALSE
+	if(!mad_fools)
+		mad_fools = rand(1, 10)
+	switch(mad_fools)
+		if(1 to 5) //hallucination
+			trait_holder.hallucination += rand(10, 30)
+		if(6) //meta warning
+			to_chat(trait_holder, "<span class='userdanger extremelybig'>[pick("Leave the server" , "Close the game window")] [pick("immediately", "right now")].</span>")
+		if(7) //supermatter explosion
+			SEND_SOUND(trait_holder, 'sound/magic/charge.ogg')
+			to_chat(trait_holder, "<span class='boldannounce'>You feel reality distort for a moment...</span>")
+		if(8) //narsie rising
+			SEND_SOUND(trait_holder, 'sound/creatures/narsie_rises.ogg')
+			to_chat(trait_holder, "<span class='narsie'>NAR-SIE HAS RISEN</span>")
+		if(9) //spooky ghost orbit
+			var/see_invis = trait_holder.see_invisible
+			notify_ghosts("[trait_holder]'s schizophrenia is making them see ghosts!",
+				enter_link = "<a href=?src=[REF(src)];orbit=1>(Click to orbit to freak them out)</a>",
+				source = trait_holder, action = NOTIFY_ORBIT)
+			sleep(50)
+			trait_holder.see_invisible = INVISIBILITY_OBSERVER
+			sleep(50)
+			notify_ghosts("[trait_holder] can no longer see ghosts.")
+			trait_holder.see_invisible = see_invis
+		if(10) //ark death
+			SEND_SOUND(trait_holder, 'sound/machines/clockcult/ark_deathrattle.ogg')
+			trait_holder.playsound_local(trait_holder, 'sound/effects/clockcult_gateway_disrupted.ogg', 50, FALSE, pressure_affected = FALSE)
+			sleep(27)
+			trait_holder.playsound_local(trait_holder, 'sound/effects/explosion_distant.ogg', 50, FALSE, pressure_affected = FALSE)
+
+/datum/trait/insanity/proc/disclaimer() //I don't /think/ we'll need this but for newbies who think "roleplay as insane" = "license to kill" it's probably a good thing to have
```

Comment:
Put it in the description, as i learned from my PR ingraining server policy in code is dumb